.github/workflows/build.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          platforms: linux/amd64,linux/arm64
```

Comment:
We might see if we can get SQuaRE to add a `platforms` argument to their https://github.com/lsst-sqre/build-and-push-to-ghcr action; that might simplify this.

Reply:
Created [build-and-push](https://github.com/lsst-dm/build-and-push) to add a few other features that we use like label, platform and build-args.